search/search_repository.go

```diff
@@ -340,10 +373,12 @@ type Query struct {
 	// an operator like "$AND", or "$OR". If the Name is not an operator, the
 	// Children slice MUST be empty.
 	Children []Query
+	// The Options represent the query options provided by the user.
+	Options *QueryOptions
 }
 
 func isOperator(str string) bool {
-	return str == AND || str == OR
+	return str == AND || str == OR || str == OPTS
```

Comment:
If you change this function, please also change the `TestIsOperator` test function in the whitebox tests.

Reply:
Done. 3b7714616807204e908ff9d68fa4d1a072d17a1d

Reply:
I've been thinking about this and I came to the conclusion that `$OPTS` isn't really an operator at all. It is just special but we shouldn't treat it as operator. Sorry for not seeing this at first.

Reply:
removed it from here. 04aa0e30c312e0f21ce1894185999e69f51beaa7